packages/client/src/schema-component/antd/association-field/InternalViewer.tsx

```
@@ -163,3 +163,41 @@ export const ReadPrettyInternalViewer = observer(
   { displayName: 'ReadPrettyInternalViewer' },
 );
+
+const customTitle = (record, schema, fieldNames) => {
+  if (schema['x-component-props']?.mode === 'CustomTitle' && schema['x-component-props']?.fieldNames?.formula) {
+    const regex = /{{(.*?)}}/g;
+    const valueOject = {};
+    let outputStr = '';
+    let match;
+    const formula = schema['x-component-props']?.fieldNames?.formula;
+    while ((match = regex.exec(formula))) {
+      if (match[1].includes('.')) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)